src/main/java/time/TimeInCapitals/showtime/LocalTimeService.java

```diff
@@ -47,27 +55,40 @@ public String getTimeAndDate() {
 		return localTimeAndDate;
 	}
 
-	public String changeTheFirstLetterToUppercase(String word) {
-
-		String[] regexResults = word.split("_");
+	public void getDataFromFile() {
 
+		File file = new File("D:/java/europe_capitals.txt");
 		StringBuilder stringBuilder = new StringBuilder();
+		String dataFromFile = null;
 
-		if (regexResults.length == 1) {
+		try (BufferedReader buffer = new BufferedReader(new FileReader(file))) {
 
-			String wordAfterChange = (stringBuilder.append(word.substring(0, 1).toUpperCase())
-					.append(word.substring(1))).toString();
-			return wordAfterChange;
-		} else {
+			while (buffer.ready()) {
 
-			String wordAfterChange = (stringBuilder.append(regexResults[0].substring(0, 1).toUpperCase())
-					.append(regexResults[0].substring(1))).append("_")
-							.append(regexResults[1].substring(0, 1).toUpperCase()).append(regexResults[1].substring(1))
-							.toString();
+				stringBuilder.append(buffer.readLine()).append(" ");
+			}
 
-			return wordAfterChange;
+		} catch (IOException e1) {
+
+			e1.printStackTrace();
 		}
 
+		dataFromFile = stringBuilder.toString();
+		regexResults = dataFromFile.split(" ");
+
+	}
+
+	public boolean checkKey(String key) {
+
+		return utcSummerCapitals.containsKey(key);
+	}
+
+	public void setSummerUTCForCapitals() {
+
+		for (int i = 1; i < regexResults.length; i += 5) {
```

Comment:
Ta metoda ma ukrytą wiedzę na temat pliku z danymi i ogólnie wygląda dość nieelegancko.

Reply:
OK, pomyślę nad tym i zaproponuję jakieś inne rozwiązanie tego.

Reply:
Do zrobienia :)

Reply:
Metoda już nie istnieje, czy chodzi może o to, żeby nie robić tego za pomocą pętli for, tylko jakoś inaczej?